examples/blog-with-comment/pages/_app.tsx

```diff
@@ -1,14 +1,15 @@
 import 'tailwindcss/tailwind.css'
 
+import type { AppProps } from 'next/app'
 import Head from 'next/head'
 import Header from '../components/header'
 import { Auth0Provider } from '@auth0/auth0-react'
 
-export default function MyApp({ Component, pageProps }) {
+export default function MyApp({ Component, pageProps }: AppProps) {
   return (
     <Auth0Provider
-      clientId={process.env.NEXT_PUBLIC_AUTH0_CLIENT_ID}
-      domain={process.env.NEXT_PUBLIC_AUTH0_DOMAIN}
+      clientId={process.env.NEXT_PUBLIC_AUTH0_CLIENT_ID!}
```

Comment:
I think in this case we could add something like an `environment.d.ts` file:

https://stackoverflow.com/a/53981706

What do you think?